pandas/tests/io/test_sql.py

```diff
@@ -704,45 +704,29 @@ def test_complex(self):
         # Complex data type should raise error
         pytest.raises(ValueError, df.to_sql, 'test_complex', self.conn)
 
-    def test_to_sql_index_label(self):
-        temp_frame = DataFrame({'col1': range(4)})
-
+    @pytest.mark.parametrize("index_name,if_exists,index_label,expected", [
```

Comment:
The `if_exists` is not relevant for the test (it was just set to re-use the same table name), so you can remove it from  the parametrization

Reply:
Thanks - yep makes sense done - @jorisvandenbossche